addon/plugins/rdfa-date-plugin/index.ts

```diff
@@ -1,20 +1,17 @@
 export const defaultDateFormats: DateFormat[] = [
   {
-    label: 'Short Date',
     key: 'short',
     dateFormat: 'dd/MM/yy',
     dateTimeFormat: 'dd/MM/yy HH:mm',
   },
   {
-    label: 'Long Date',
     key: 'long',
     dateFormat: 'EEEE dd MMMM yyyy',
     dateTimeFormat: 'PPPPp',
   },
 ];
 
 export type DateFormat = {
-  label: string;
```

Comment:
I'd like to keep the option though, so instead of just removing support for the label, lets just make it optional